### ISSUE TITLE
Impossible to create an Image element with no alt text

### ISSUE THREAD

**Issue by archiecobbs** [vaadin/flow-components#2997]
### Description

I want to create an `Image` without any `alt` text.

### Expected outcome

Either a constructor like `new Image(src)` to exist, or the constructor `new Image(src, alt)` to work when `alt` is null.

### Actual outcome

The only option is `new Image(src, null)` but that fails:
```
java.lang.IllegalArgumentException: Cannot set alt to null
	at com.vaadin.flow.component.PropertyDescriptors$PropertyDescriptorImpl.set(PropertyDescriptors.java:67)
	at com.vaadin.flow.component.PropertyDescriptor.set(PropertyDescriptor.java:53)
	at com.vaadin.flow.component.Component.set(Component.java:482)
	at com.vaadin.flow.component.html.Image.setAlt(Image.java:120)
	at com.vaadin.flow.component.html.Image.<init>(Image.java:65)
```

Yes I know that `new Image(src, "")` works, but that's saying "create an image with empty string alt text", not "create an image with no alt text".

The fact that those end up being the same thing at the HTML level is not relevant; this is the Java level we're talking about here (in fact, by this logic `new Image(src, "")` should throw an `IllegalArgumentException` since empty string alt text is not representable in HTML).

### Minimal reproducible example

`new Image("abc", null)`

### Steps to reproduce

`new Image("abc", null)`

### Environment

Vaadin 22.0.10

### Browsers Affected

All.

**Recommendation:**  Update `Image` constructors to accept null, replacing it with empty string.
